src/test/ui/lint/rfc-2383-lint-reason/expect_tool_lint_rfc_2383.rs

```diff
@@ -0,0 +1,154 @@
+// check-pass
+#![feature(lint_reasons)]
+
+//! This file tests the expect attribute implementation for tool lints. The same
+//! file will be used to test clippy and rustdoc.
```

Comment:
It might be good to mention that this file is copied in three different places and we should try to keep them in sync with each other (and mention this in the other files as well).

Reply:
Good point, I thought that it was part of the comment, but it was missing now that you've mentioned it. :upside_down_face: